pkg/types/interval.go

```diff
@@ -89,6 +98,36 @@ func ParseInterval(input Interval) int {
 	return t
 }
 
+func ParseIntervalSeconds(input Interval) int {
```

Comment:
Should provide ParseInterval instead of ParseIntervalSeconds.
All result from now on should be based on Seconds.